controller/twilio.go

```diff
@@ -0,0 +1,186 @@
+package controller
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"bitbucket.org/ckvist/twilio/twiml"
```

Comment:
I saw a blurb about a new built-in replacement for godep in 1.5b1.  Maybe we should wait for that.